OrbitGl/CaptureWindow.cpp

```diff
@@ -637,6 +639,7 @@ void CaptureWindow::ResetHoverTimer() {
 
 //-----------------------------------------------------------------------------
 void CaptureWindow::Draw() {
+  auto start = std::chrono::steady_clock::now();
```

Comment:
There's also an `OrbitTicks` in `OrbitCore/Profiling.h`, used a few times in the code. I don't prefer it, just to let you know.

Reply:
Removed instrumentation as we don't really need it.